### DISCUSSION TITLE
How to re-activate comments?

### DISCUSSION THREAD

**Discussion by mozonte** [datenstrom/community#445] [Ask a question] [answered]
How do you make comments active even on posts that are already published before installation?  
When I installed, comments worked on new posts but were 'closed' on all older posts. I would like comments to always be open on all posts (unless manually closed).

**Comment by schulle4u** [comment, marked as answer]
According to [the extension's documentation](https://github.com/GiovanniSalmeri/yellow-comments), you can set `commentsOpening: 0` in your `system/settings/system.ini`. This should leave comment sections for all blog pages opened, and re-open already closed comments. Maybe @GiovanniSalmeri can answer further questions.

**Comment by mozonte** [comment]
Thanks @schulle4u 

That did it.

I had been trying to work in the /system/extensions/yellow-comments-master/comments.php files (which have the same fields) but your solution did it. 

Resolved.
